### ISSUE TITLE
Multibase scale bug

### ISSUE THREAD

**Issue by equivalentideas** [modularscale/modularscale-sass#75]
Using scale:

```
$ms-base: 1rem 1.0625rem;
$ms-ratio: $golden;
```

@debug ms-list(0, 2, $ms-base, $ms-ratio);

returns:

```
DEBUG: 1rem 1.0625rem 1.61803rem
```

but when calling scale 1 with:

```
font-size: ms(1);
```

The processed CSS appears as:

```
font-size: 1.61803rem;
```

In other words it's skipping to scale 2.

**Comment by DornieDortsch** [comment]
The bug still exists on negative values

```
$ms-base: 1em 3.75em;
$ms-ratio: $fourth;
```

@debug ms-list(-2, 0, $ms-base, $ms-ratio);

returns:

```
DEBUG: 0.75em 0.88989em 1em
```

but when calling scale -1 with:

```
font-size: ms(-1);
```

The processed CSS appears as:

```
font-size: 0.75em
```

Same behavior, it's skipping to scale -2.

**Comment by scottkellum** [comment]
@equivalentideas, @DornieDortsch: How are you running modular scale? What version of Sass/libsass? Installed via Compass or another method?

**Comment by scottkellum** [comment]
unable to replicate in any version: http://sassmeister.com/gist/10592343

**Comment by equivalentideas** [comment]
Sorry I should have closed this, I updated and this was no longer occurring. My bad, thanks @scottkellum :)

**Comment by DornieDortsch** [comment]
Sorry for the late reply, but I run in the same error again. Negative values and multibase.

http://sassmeister.com/gist/41dbd8c5eccbf67de20a

**Comment by scottkellum** [comment]
reopening

**Comment by scottkellum** [comment]
This will be solved in 3.0.
